NumRush.py

```diff
@@ -363,6 +364,21 @@ def run(self):
             pygame.display.update()
             fpsClock.tick(FPS)
 
+    def restore_game(self, high_score=0, current_score=0, game=0):
+        ''' Restore a game from the Journal '''
+        # TODO: Save/restore recall list for game 2
+        hscore = high_score
+        score = current_score
+        self.game = game
+        for i, dot in enumerate(dot_list):
```

Comment:
`dot_list` is used here but I don't see where it's defined or passed as an arg.

Reply:
I think you might have reviewed older commit, I've fixed it in later commit